op-devstack/sysgo/superroot.go

```diff
@@ -225,10 +228,38 @@ func getSuperRoot(t devtest.CommonT, o *Orchestrator, timestamp uint64, supervis
 	return super.SuperRoot
 }
 
-func getInteropAbsolutePrestate(t devtest.CommonT) common.Hash {
+func getInteropCannonAbsolutePrestate(t devtest.CommonT) common.Hash {
 	return getAbsolutePrestate(t, "op-program/bin/prestate-proof-interop.json")
 }
 
+func getInteropCannonKonaAbsolutePrestate(t devtest.CommonT) common.Hash {
+	return common.HexToHash(findKonaVariable(t, "KONA_INTEROP_PRESTATE_HASH"))
+}
+
+func getCannonKonaAbsolutePrestate(t devtest.CommonT) common.Hash {
+	return common.HexToHash(findKonaVariable(t, "KONA_PRESTATE_HASH"))
+}
+
+func findKonaVariable(t devtest.CommonT, name string) string {
+	konaJustfilePath := "kona/justfile"
```

Comment:
nit: Rather than parsing the justfile, the kona variables could be stored in a "common" format. JSON or whatever. That both the justfile and sysgo can consume. This is good enough though.

Reply:
Yeah I considered that but having the justfile be completely standalone is quite nice and makes the caching work easily.